### DISCUSSION TITLE
Fan control impact on performance?

### DISCUSSION THREAD

**Discussion by vertexgamer** [Rem0o/FanControl.Releases#2334] [Q&A] [answered]
Has anyone measured the impact on performance that fan control causes?
Also in case the OS will stop responding will the fans keep the speed set by fan control or they will default to whatever the bios specify?
I often find myself running multithreaded workloads that uses almost all of my physical memory, as you can imagine in a possibility of fan control failing to increase the fan speed during a heavy load, the cpu will thermal throttle pretty fast and shut itself off, which will cause me to lose all progress.

**Comment by Rem0o** [comment, marked as answer]
In extreme scenarios, think synthetic loads, fan control "may" slow down if the OS scheduler doesn't gives it any CPU time, but that's the same with any program.   Given that scenario, the fans would be stuck at the latest command sent, so whatever they were spinning at.
